config/software/ncurses.rb

```diff
@@ -64,6 +64,14 @@
     patch source: "ncurses-5.9-solaris-xopen_source_extended-detection.patch", plevel: 0
   end
 
+  # Update config.guess to support platforms made after 2010 (like aarch64)
+  patch source: "config_guess_2015-09-24.patch", plevel: 0
+
+  # Patch to add support for GCC 5, doesn't break previous versions
+  if version == "5.9"
+    patch source: "ncurses-5.9-gcc-5.patch", plevel: 1
+  end
+
```

Comment:
These patches conflict with the ``v5.9.ppc64le-configure.patch`` patch below. Removing the one below seems to be the right method to work around it.